src/common/config.cc

```diff
@@ -1227,6 +1227,47 @@ void md_config_t::diff(
   }
 }
 
+void md_config_t::diff_setting(const md_config_t *other,
+            std::map<std::string, std::pair<std::string, std::string>> *diff, 
+            std::set<std::string> *unknown, const std::string& ceph_setting, bool show_unchanged)
+{
+  Mutex::Locker l(lock);
+
+  bool wasItFound = false;
```

Comment:
our naming convention does not encourage `wasItFound`, might want to use `hello_world` instead.